### ISSUE TITLE
Loader app not showing up after "successful" jailbreak

### ISSUE THREAD

**Issue by CobreDev** [checkra1n/BugTracker#1810]
**Tell us about your setup:**
1. What iDevice are you using? **iPhone 7**
2. On what version of iOS is it? **iOS 14.1**
3. What version of checkra1n are you using? **beta 0.12.0**
4. What is your host system (OS version? Hackintosh? VM? etc.)? **2012 MacBook Air running macOS 10.14.6 Mojave**
5. How are you connecting to the device (USB-A? USB-C? Apple/3rd party cable? Through a USB hub?)? **First-Party Apple USB-A to Lightning cable**

**What are the steps to reproduce the issue?**
1. Plug phone in
2. Run checkra1n
3. Fail to find Loader app
...

**What do you expect, and what is happening instead?**
I expect the Loader app to show up on my phone, but it's been ~8 hours and nothing has showed up.

**Does the issue also occur if you tick "Safe Mode" in the checkra1n options?**
It doesn't matter if I use Safe Mode or not

**Any other info, error logs, screenshots, ...?**
No logs, checkra1n seems to jailbreak fine.

I was able to SSH into my phone, and I've ran `uicache -a`, `killall SpringBoard`, and `killall -9 backboardd` multiple times without success.

**Comment by fennectech** [comment]
Im getting the same thign with an iPad pro   get USB error code -76

**Comment by CobreDev** [comment]
I’d say that’s a different issue, since I’m not getting any errors

**Comment by wolfiediscord** [comment]
Can Verify.

My System:
- Device: iPhone 7 (iOS 14.0)
- Host: Debian 10
- Version: beta 0.12.0
- Cable: Third Party (not the issue)

However, when I did `checkra1n -cv` in the terminal after entering DFU mode, it appears to be fine except for a libusb error.
```
<Error> libusb: Failed to get active config descriptor: LIBUSB_ERROR_NOT_FOUND
```
I also cannot SSH into the device at all.

I hope this issue gets fixed soon. I would love to jailbreak my iPhone.

EDIT: Updating to 14.1 doesn't seem to fix the issue either.
EDIT 2: I also tried on an iMac running Mojave, but it seems to not work either.

**Comment by 01M** [comment]
same

**Comment by angelobdev** [comment]
Same here on iPhone X (GSM) 🤦🏻‍♂️

**Comment by wolfiediscord** [comment]
Huge thanks to @CobreDev for linking this post in a discord server. [This post is a workaround until this bug is fixed.](https://www.reddit.com/r/jailbreak/comments/jqqyhu/tutorial_how_to_fix_checkra1n_0120_app_not/)

**Comment by CobreDev** [comment]
Yeah this seems to work, but it requires you to use OdysseyRa1n, which some people might not like

**Comment by alphabt** [comment]
Found a workaround: open Shortcuts app and create a shortcut to launch Checkra1n. I had to do the same thing after installing Cydia.

**Comment by CobreDev** [comment]
Hmm that’s a good idea, thanks
